extensions/BMO/Extension.pm

```diff
@@ -713,6 +713,18 @@ sub bug_format_comment {
         }
     });
 
+    # link github pull requests and issues
+    push (@$regexes, {
+        match => qr#\b([\S]+\/[\S]+\#[0-9]+)\b#mx,
```

Comment:
As far as I can see, the repo name can only contains `[A-Za-z0-9_\.-]`. I guess the owner part is that way as well. Actually `[\S]` looks wrong anyway, since it could include `/` itself, right?

Also I think you may want to ensure that the character before is not a `/` either to avoid parsing part of a URL.